app/src/components/widgets/multiLevelWidgets/cumulativeTrendChart/chartjsConfig.js

```diff
@@ -23,8 +23,8 @@ const Color = require('color');
 const getDefects = (fields) => fields.filter((item) => /defects/.test(item));
 
 const getExecutions = () => [
-  'statistics$executions$passed',
   'statistics$executions$failed',
+  'statistics$executions$passed',
   'statistics$executions$skipped',
 ];
 
```

Comment:
Also please clarify the order of the legend items with BA (see the screenshot below), because it is also controlled by the `executions` order.

![image](https://user-images.githubusercontent.com/31861805/99374263-7620be80-28d3-11eb-8107-099c6a415f29.png)

Reply:
BA is ok with this, add a description in the subtask